src/modules/reports/components/ReportsFetcher/ReportsFetcher.jsx

```diff
@@ -5,7 +5,12 @@ import { Loader } from 'modules/ui';
 import { useReportsFetcher } from 'modules/reports';
 
 const ReportsFetcher = ({ children }) => {
-    const { success, inProgress } = useReportsFetcher();
+    const defaultParams = {
+        appName: 'All',
```

Comment:
this All should be constant and also translated

Reply:
done